From: Nicolas Dandrimont <nicolas.dandrimont@crans.org>
Date: Wed, 4 Dec 2013 22:15:52 +0100
Subject: Link boost libraries without the mt suffix

---
 snapper/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/snapper/Makefile.am b/snapper/Makefile.am
index a4a8c60..23ac5bc 100644
--- a/snapper/Makefile.am
+++ b/snapper/Makefile.am
@@ -53,7 +53,7 @@ endif
 
 
 libsnapper_la_LDFLAGS = -version-info @LIBVERSION_INFO@
-libsnapper_la_LIBADD = -lboost_thread-mt -lboost_system-mt -lxml2 -lz -lm
+libsnapper_la_LIBADD = -lboost_thread -lboost_system -lxml2 -lz -lm
 
 pkgincludedir = $(includedir)/snapper
 
