From: Hideki Yamane <henrich@debian.org>
Date: Sun, 19 Feb 2017 23:33:05 +0900
Subject: Bug#852574 Update udevadm path to /bin/udevadm

---
 client/mksubvolume.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: snapper/client/mksubvolume.cc
===================================================================
--- snapper.orig/client/mksubvolume.cc
+++ snapper/client/mksubvolume.cc
@@ -109,7 +109,7 @@ do_tmp_umount(const libmnt_fs* fs, const
     if (verbose)
 	cout << "do-tmp-umount" << endl;
 
-    system("/sbin/udevadm settle --timeout 20");
+    system("/bin/udevadm settle --timeout 20");
 
     libmnt_fs* x = mnt_copy_fs(NULL, fs);
     if (!x)
