From: Hideki Yamane <henrich@debian.org>
Date: Fri, 6 Apr 2018 12:06:55 +0900
Subject: Add debian specific option DISABLE_APT_SNAPSHOT

---
 data/sysconfig.snapper | 2 ++
 1 file changed, 2 insertions(+)

Index: snapper/data/sysconfig.snapper
===================================================================
--- snapper.orig/data/sysconfig.snapper
+++ snapper/data/sysconfig.snapper
@@ -5,3 +5,5 @@
 # List of snapper configurations.
 SNAPPER_CONFIGS=""
 
+# if you want to disable snapshot per install/upgrade, then set "yes"
+DISABLE_APT_SNAPSHOT="no"
