From: Hideki Yamane <henrich@debian.org>
Date: Sun, 15 Aug 2021 14:56:57 +0900
Subject: Add some missing systemd unit directives

---
 data/boot.service     | 1 +
 data/snapperd.service | 3 +++
 2 files changed, 4 insertions(+)

diff --git a/data/boot.service b/data/boot.service
index da5966e..3d9fc75 100644
--- a/data/boot.service
+++ b/data/boot.service
@@ -1,6 +1,7 @@
 [Unit]
 Description=Take snapper snapshot of root on boot
 ConditionPathExists=/etc/snapper/configs/root
+Documentation=man:snapper(8)
 
 [Service]
 Type=oneshot
diff --git a/data/snapperd.service b/data/snapperd.service
index cad1620..15f5db8 100644
--- a/data/snapperd.service
+++ b/data/snapperd.service
@@ -6,3 +6,6 @@ Documentation=man:snapperd(8)
 Type=dbus
 BusName=org.opensuse.Snapper
 ExecStart=/usr/sbin/snapperd
+
+[Install]
+WantedBy=sysinit.target
