From: Hideki Yamane <henrich@debian.org>
Date: Tue, 10 Dec 2019 22:28:54 +0900
Subject: drop testsuite-real to avoid to be installed into package

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: snapper/Makefile.am
===================================================================
--- snapper.orig/Makefile.am
+++ snapper/Makefile.am
@@ -3,7 +3,7 @@
 #
 
 SUBDIRS = snapper examples dbus server client scripts pam data doc po		\
-	testsuite testsuite-real testsuite-cmp zypp-plugin
+	testsuite testsuite-cmp zypp-plugin
 
 AUTOMAKE_OPTIONS = foreign dist-bzip2 no-dist-gzip
 
