From 84507975ab73229b8f9d798aa93515685831deab Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Fri, 14 Nov 2025 12:13:38 +0100
Subject: [PATCH 08/74] require dbus-service

---
 snapper.spec.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/snapper.spec.in b/snapper.spec.in
index ca780e94..ef29dca1 100644
--- a/snapper.spec.in
+++ b/snapper.spec.in
@@ -106,6 +106,7 @@ BuildRequires:  zlib-devel
 %if %{with coverage}
 BuildRequires:  lcov
 %endif
+Requires:       dbus-service
 Requires:       diffutils
 Requires:       libsnapper@LIBVERSION_MAJOR@ = %version
 Requires:       systemd
-- 
2.43.0

