From 9c136e78b5c265c36b7bb192332bd5a1acfcb1c2 Mon Sep 17 00:00:00 2001
From: Cheng-Ling Lai <jameslai.tech@gmail.com>
Date: Fri, 24 Apr 2026 16:15:56 +0800
Subject: [PATCH 10/65] Fix snapper-sync installation

---
 scripts/Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/scripts/Makefile.am b/scripts/Makefile.am
index ecdd04b8..714cec4f 100644
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -4,6 +4,8 @@
 
 SUBDIRS = completion
 
+sbin_SCRIPTS = snapper-sync
+
 if HAVE_PAM
 
 pam_snapperdir = /usr/lib/pam_snapper
@@ -20,4 +22,3 @@ EXTRA_DIST = snapper-hourly snapper-sync $(pam_snapper_SCRIPTS)
 
 install-data-local:
 	install -D snapper-hourly $(DESTDIR)/etc/cron.hourly/suse.de-snapper
-	install -D snapper-sync $(DESTDIR)/usr/sbin
-- 
2.43.0

