From 3fd3932a81ee9dad191220d0b2f4de553a2ede97 Mon Sep 17 00:00:00 2001
From: Cheng-Ling Lai <jameslai.tech@gmail.com>
Date: Mon, 27 Apr 2026 09:33:11 +0800
Subject: [PATCH 11/65] Fix snapper.spec.in for systemd override directories

---
 snapper.spec.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/snapper.spec.in b/snapper.spec.in
index c0f1e4b7..b6974acd 100644
--- a/snapper.spec.in
+++ b/snapper.spec.in
@@ -375,7 +375,9 @@ A backup program for snapshots created by snapper.
 %dir %{_sysconfdir}/snapper/certs
 %{_unitdir}/snapper-{backup}.*
 %{_unitdir}/snapper-sync.service
+%{_unitdir}/snapper-boot.timer.d
 %{_unitdir}/snapper-boot.timer.d/10-snapper-sync-override.conf
+%{_unitdir}/snapper-timeline.timer.d
 %{_unitdir}/snapper-timeline.timer.d/10-snapper-sync-override.conf
 %{_mandir}/*/snbk.8*
 %{_mandir}/*/snapper-backup-configs.5*
-- 
2.43.0

