From dbbdc79e35603ef56417c7f13fcfdebd7cc10607 Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Tue, 12 May 2026 11:17:09 +0200
Subject: [PATCH 12/65] - added Ubuntu 26.04

---
 Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 81349f33..5daf7bdc 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -31,7 +31,8 @@ UBUNTU_FLAVOURS =	\
 	xUbuntu_24.04	\
 	xUbuntu_24.10	\
 	xUbuntu_25.04	\
-	xUbuntu_25.10
+	xUbuntu_25.10	\
+	xUbuntu_26.04
 
 RASPBIAN_FLAVOURS =	\
 	Raspbian_11	\
-- 
2.43.0

