From 173c90a64b2f7bb3a0c4ad76ad646dd9bc3a1c0e Mon Sep 17 00:00:00 2001
From: Cheng-Ling Lai <jamesljlster@gmail.com>
Date: Fri, 12 Dec 2025 11:08:25 +0800
Subject: [PATCH 25/74] Added a notice to snbk restore help

---
 client/snbk/cmd-restore.cc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/client/snbk/cmd-restore.cc b/client/snbk/cmd-restore.cc
index 78a4e2e8..7f3ede45 100644
--- a/client/snbk/cmd-restore.cc
+++ b/client/snbk/cmd-restore.cc
@@ -45,6 +45,9 @@ namespace snapper
     {
 	cout << "  " << _("Restore:") << '\n'
 	     << "\t" << _("snbk restore [numbers]") << '\n'
+	     << '\n'
+	     << "\t" << _("Notice: Restart the `snapperd` service to make the restored "
+			  "snapshots visible immediately.") << '\n'
 	     << '\n';
     }
 
-- 
2.43.0

