From aee4d5065e4ae224ca8085cabb0d8765a89fae1a Mon Sep 17 00:00:00 2001
From: Cheng-Ling Lai <jamesljlster@gmail.com>
Date: Fri, 12 Dec 2025 21:23:55 +0800
Subject: [PATCH 29/98] Added snbk restore to bash completion

---
 scripts/completion/snbk-completion.bash | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/completion/snbk-completion.bash b/scripts/completion/snbk-completion.bash
index cc8d6744..5a8b4969 100644
--- a/scripts/completion/snbk-completion.bash
+++ b/scripts/completion/snbk-completion.bash
@@ -30,6 +30,7 @@ _snbk()
         "list-configs"
         "list" "ls"
         "transfer"
+	"restore"
 	"delete"
 	"transfer-and-delete")
 
-- 
2.43.0

