From d1fa14beee23114445a91371200262a86054e982 Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Thu, 18 Dec 2025 10:08:01 +0100
Subject: [PATCH 33/74] - target state must be valid for restore

---
 client/snbk/TheBigThing.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/snbk/TheBigThing.cc b/client/snbk/TheBigThing.cc
index b139f97a..a61ac4fd 100644
--- a/client/snbk/TheBigThing.cc
+++ b/client/snbk/TheBigThing.cc
@@ -183,7 +183,7 @@ namespace snapper
 	if (!quiet)
 	    cout << sformat(_("Restoring snapshot %d."), num) << '\n';
 
-	if (target_state == TargetState::MISSING)
+	if (target_state != TargetState::VALID)
 	    SN_THROW(Exception(_("Snapshot not on target.")));
 
 	if (source_state != SourceState::MISSING)
-- 
2.43.0

