From bb3cbba0df587e5e1fdd3529d6badc2b9e47abe5 Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Mon, 8 Jun 2026 11:11:11 +0200
Subject: [PATCH 36/65] - fixed typo

---
 dbus/DBusConnection.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dbus/DBusConnection.cc b/dbus/DBusConnection.cc
index 70f49628..a2916522 100644
--- a/dbus/DBusConnection.cc
+++ b/dbus/DBusConnection.cc
@@ -155,7 +155,7 @@ namespace DBus
     {
 	boost::lock_guard<boost::mutex> lock(mutex);
 
-	const string sender = msg.get_sender();
+	const string sender = m.get_sender();
 	if (sender.empty())
 	{
 	    SN_THROW(FatalException());
-- 
2.43.0

