From 1ccc85360d20ff0fbdecf846f88cd74cb4df8cfa Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Tue, 9 Jun 2026 10:17:01 +0200
Subject: [PATCH 37/65] - extended documentation

---
 doc/snapper.xml.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/doc/snapper.xml.in b/doc/snapper.xml.in
index ae4bd8ba..dc23de57 100644
--- a/doc/snapper.xml.in
+++ b/doc/snapper.xml.in
@@ -991,6 +991,8 @@
     </variablelist>
     <para>More actions and arguments can be added any time. Using snapper in
     the plugins is not allowed.</para>
+    <para>Ensure plugins execute within a few seconds. Slow-running plugins
+    block the primary interface and degrade user experience.</para>
     <para>It is undefined whether the plugins are called from the
     client (snapper) or server (snapperd).</para>
   </refsect1>
-- 
2.43.0

