From 80ebba0b30620343141c3215cc11e7516322263a Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Wed, 10 Jun 2026 09:02:25 +0200
Subject: [PATCH 38/65] - reply with "no permissions" if uid detection fails

---
 server/snapperd.cc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/server/snapperd.cc b/server/snapperd.cc
index cb642b1c..d8091f1c 100644
--- a/server/snapperd.cc
+++ b/server/snapperd.cc
@@ -124,6 +124,10 @@ MyMainLoop::method_call(DBus::Message& msg)
 	    {
 		SN_CAUGHT(e);
 		y2err("failed to get uid of client");
+
+		DBus::MessageError reply(msg, "error.no_permissions", DBUS_ERROR_FAILED);
+		send(reply);
+
 		return;
 	    }
 
-- 
2.43.0

