From 8c0143f4dd0675914a0703fa47c74e738330ff1d Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Wed, 10 Jun 2026 09:14:14 +0200
Subject: [PATCH 39/65] - add changelog entry

---
 package/snapper.changes | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/package/snapper.changes b/package/snapper.changes
index 291a0245..3a3a702b 100644
--- a/package/snapper.changes
+++ b/package/snapper.changes
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Jun 10 09:13:13 CEST 2026 - Arvin Schnell <aschnell@suse.com>
+
+- improve error handling if uid of client cannot be detected
+  (see bsc#1265853)
+
 -------------------------------------------------------------------
 Fri 24 Apr 05:05:24 UTC 2026 - Cheng-Ling Lai <jameslai.tech@gmail.com>
 
-- 
2.43.0

