From 17686118aecf4873be09a440286eddb8b44953cf Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Thu, 18 Jun 2026 12:21:55 +0200
Subject: [PATCH 45/65] - extended documentation

---
 doc/snapper.xml.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/doc/snapper.xml.in b/doc/snapper.xml.in
index dc23de57..97d634de 100644
--- a/doc/snapper.xml.in
+++ b/doc/snapper.xml.in
@@ -992,7 +992,8 @@
     <para>More actions and arguments can be added any time. Using snapper in
     the plugins is not allowed.</para>
     <para>Ensure plugins execute within a few seconds. Slow-running plugins
-    block the primary interface and degrade user experience.</para>
+    block the primary interface and degrade user experience and can even
+    cause timeouts (e.g. causing zypper to kill the snapper plugin).</para>
     <para>It is undefined whether the plugins are called from the
     client (snapper) or server (snapperd).</para>
   </refsect1>
-- 
2.43.0

