From 8a66bbfe557551441ea8dec95e12d101e7bd970e Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Tue, 23 Jun 2026 08:21:36 +0200
Subject: [PATCH 54/65] - fixed memory leak in error case

---
 client/proxy/commands.cc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/client/proxy/commands.cc b/client/proxy/commands.cc
index 699ba529..ef66e461 100644
--- a/client/proxy/commands.cc
+++ b/client/proxy/commands.cc
@@ -569,6 +569,7 @@ command_get_xfiles_by_pipe(DBus::Connection& conn, const string& config_name, un
 	    if (fin.eof() != 0)
 		break;
 
+	    free(buffer);
 	    SN_THROW(IOErrorException("reading pipe failed, getline failed: " + stringerror(errno)));
 	}
 
@@ -579,7 +580,10 @@ command_get_xfiles_by_pipe(DBus::Connection& conn, const string& config_name, un
 
 	string::size_type pos1 = line.find(" ");
 	if (pos1 == string::npos)
+	{
+	    free(buffer);
 	    SN_THROW(IOErrorException("reading pipe failed, parse error"));
+	}
 
 	string::size_type pos2 = line.find(" ", pos1 + 1);
 
-- 
2.43.0

