From 4f872c36aee469f0059d6ddb4b710c25d3b10b9c Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Tue, 30 Jun 2026 14:16:27 +0200
Subject: [PATCH 60/65] - improved error handling

---
 snapper/Compare.cc | 23 ++++++++++++++++-------
 1 file changed, 16 insertions(+), 7 deletions(-)

diff --git a/snapper/Compare.cc b/snapper/Compare.cc
index 7ea002a1..3c441219 100644
--- a/snapper/Compare.cc
+++ b/snapper/Compare.cc
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) [2011-2013] Novell, Inc.
+ * Copyright (c) 2026 SUSE LLC
  *
  * All Rights Reserved.
  *
@@ -279,7 +280,9 @@ namespace snapper
 	    cb(path + "/" + *it, status);
 
 	    struct stat stat;
-	    dir.stat(*it, &stat, AT_SYMLINK_NOFOLLOW);
+	    if (dir.stat(*it, &stat, AT_SYMLINK_NOFOLLOW) != 0)
+		SN_THROW(IOErrorException("stat failed path: " + dir.fullname() + "/" + *it));
+
 	    if (S_ISDIR(stat.st_mode))
 		listSubdirs(SDir(dir, *it), path + "/" + *it, status, cb);
 	}
@@ -370,7 +373,8 @@ namespace snapper
 	    else if (first1 == last1)
 	    {
 		struct stat stat2;
-		dir2.stat(*first2, &stat2, AT_SYMLINK_NOFOLLOW); // TODO error check
+		if (dir2.stat(*first2, &stat2, AT_SYMLINK_NOFOLLOW) != 0)
+		    SN_THROW(IOErrorException("stat failed path: " + dir2.fullname() + "/" + *first2));
 
 		if (stat2.st_dev == cmp_data.dev2)
 		    lonesome(dir2, path, *first2, stat2, CREATED, cmp_data.cb);
@@ -380,7 +384,8 @@ namespace snapper
 	    else if (first2 == last2)
 	    {
 		struct stat stat1;
-		dir1.stat(*first1, &stat1, AT_SYMLINK_NOFOLLOW); // TODO error check
+		if (dir1.stat(*first1, &stat1, AT_SYMLINK_NOFOLLOW) != 0)
+		    SN_THROW(IOErrorException("stat failed path: " + dir1.fullname() + "/" + *first1));
 
 		if (stat1.st_dev == cmp_data.dev1)
 		    lonesome(dir1, path, *first1, stat1, DELETED, cmp_data.cb);
@@ -390,7 +395,8 @@ namespace snapper
 	    else if (*first2 < *first1)
 	    {
 		struct stat stat2;
-		dir2.stat(*first2, &stat2, AT_SYMLINK_NOFOLLOW); // TODO error check
+		if (dir2.stat(*first2, &stat2, AT_SYMLINK_NOFOLLOW) != 0)
+		    SN_THROW(IOErrorException("stat failed path: " + dir2.fullname() + "/" + *first2));
 
 		if (stat2.st_dev == cmp_data.dev2)
 		    lonesome(dir2, path, *first2, stat2, CREATED, cmp_data.cb);
@@ -400,7 +406,8 @@ namespace snapper
 	    else if (*first1 < *first2)
 	    {
 		struct stat stat1;
-		dir1.stat(*first1, &stat1, AT_SYMLINK_NOFOLLOW); // TODO error check
+		if (dir1.stat(*first1, &stat1, AT_SYMLINK_NOFOLLOW) != 0)
+		    SN_THROW(IOErrorException("stat failed path: " + dir1.fullname() + "/" + *first1));
 
 		if (stat1.st_dev == cmp_data.dev1)
 		    lonesome(dir1, path, *first1, stat1, DELETED, cmp_data.cb);
@@ -413,10 +420,12 @@ namespace snapper
 		    SN_THROW(LogicErrorException());
 
 		struct stat stat1;
-		dir1.stat(*first1, &stat1, AT_SYMLINK_NOFOLLOW); // TODO error check
+		if (dir1.stat(*first1, &stat1, AT_SYMLINK_NOFOLLOW) != 0)
+		    SN_THROW(IOErrorException("stat failed path: " + dir1.fullname() + "/" + *first1));
 
 		struct stat stat2;
-		dir2.stat(*first2, &stat2, AT_SYMLINK_NOFOLLOW); // TODO error check
+		if (dir2.stat(*first2, &stat2, AT_SYMLINK_NOFOLLOW) != 0)
+		    SN_THROW(IOErrorException("stat failed path: " + dir2.fullname() + "/" + *first2));
 
 		twosome(cmp_data, dir1, dir2, path, *first1, stat1, stat2);
 		++first1;
-- 
2.43.0

