From 1cc57c8b60eede8e7b4e837fc171dc704368f5ef Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Tue, 30 Jun 2026 17:46:25 +0200
Subject: [PATCH 64/65] - fixed error message

---
 snapper/Compare.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/snapper/Compare.cc b/snapper/Compare.cc
index a6333dbd..9608b780 100644
--- a/snapper/Compare.cc
+++ b/snapper/Compare.cc
@@ -252,7 +252,7 @@ namespace snapper
 	    SN_THROW(IOErrorException("stat failed path:" + file1.fullname()));
 
 	if (r2 != 0)
-	    SN_THROW(IOErrorException("lstat failed path:" + file2.fullname()));
+	    SN_THROW(IOErrorException("stat failed path:" + file2.fullname()));
 
 	return cmpFiles(file1, stat1, file2, stat2);
     }
-- 
2.43.0

