From 1c02e683ed17b83ed64f12e28a387b6b541d3d5c Mon Sep 17 00:00:00 2001
From: Cheng-Ling Lai <jameslai.tech@gmail.com>
Date: Mon, 16 Feb 2026 15:59:27 +0800
Subject: [PATCH 85/98] Rearrange precondition checks for snapshot restore

---
 client/snbk/TheBigThing.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/client/snbk/TheBigThing.cc b/client/snbk/TheBigThing.cc
index 812cf807..4d20e5c6 100644
--- a/client/snbk/TheBigThing.cc
+++ b/client/snbk/TheBigThing.cc
@@ -302,12 +302,12 @@ namespace snapper
 	if (!quiet)
 	    cout << sformat(_("Restoring snapshot %d."), num) << '\n';
 
-	if (target_state != TargetState::VALID)
-	    SN_THROW(Exception(_("Snapshot not on target.")));
-
 	if (source_state != SourceState::MISSING)
 	    SN_THROW(Exception(_("Snapshot already on source.")));
 
+	if (target_state != TargetState::VALID)
+	    SN_THROW(Exception(_("Snapshot not on target.")));
+
 	// Copy the snapshot from the target to the source
 	copy(backup_config, the_big_things,
 	     make_copy_specs(backup_config, the_big_things, CopyMode::TARGET_TO_SOURCE));
-- 
2.43.0

