From 4eda3e60480ed89c3c971c2e7c54885bb392d50f Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.de>
Date: Tue, 17 Mar 2026 11:10:32 +0100
Subject: [PATCH 87/98] - coding style

---
 client/snapper/cmd-list.cc | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/client/snapper/cmd-list.cc b/client/snapper/cmd-list.cc
index 9ca01548..7bc0e9e5 100644
--- a/client/snapper/cmd-list.cc
+++ b/client/snapper/cmd-list.cc
@@ -135,9 +135,10 @@ namespace snapper
 
 	private:
 
+	    const string root_prefix;
+
 	    ProxySnapshots::const_iterator default_snapshot;
 	    ProxySnapshots::const_iterator active_snapshot;
-	    string root_prefix;
 
 	    bool used_space_broken = true;
 
@@ -164,8 +165,8 @@ namespace snapper
 	OutputHelper::OutputHelper(const ProxySnapper* snapper, const vector<Column>& columns,
 				   const string& root_prefix)
 	    : snapper(snapper), locker(snapper), snapshots(snapper->getSnapshots()),
-	      default_snapshot(snapshots.end()), active_snapshot(snapshots.end()),
-	      root_prefix(root_prefix)
+	      root_prefix(root_prefix), default_snapshot(snapshots.end()),
+	      active_snapshot(snapshots.end())
 	{
 	    try
 	    {
-- 
2.43.0

