From: Hideki Yamane <henrich@debian.org>
Date: Sun, 15 Aug 2021 14:56:57 +0900
Subject: Add some missing systemd unit directives

---
 data/boot.service     | 1 +
 data/snapperd.service | 3 +++
 2 files changed, 4 insertions(+)

Index: snapper/data/boot.service
===================================================================
--- snapper.orig/data/boot.service
+++ snapper/data/boot.service
@@ -5,6 +5,7 @@ Requires=dbus.service
 After=dbus.service
 Requires=nss-user-lookup.target
 After=nss-user-lookup.target
+Documentation=man:snapper(8)
 
 [Service]
 Type=oneshot
Index: snapper/data/snapperd.service
===================================================================
--- snapper.orig/data/snapperd.service
+++ snapper/data/snapperd.service
@@ -15,3 +15,6 @@ ProtectHostname=true
 RestrictAddressFamilies=AF_UNIX
 RestrictRealtime=true
 PrivateMounts=no
+
+[Install]
+WantedBy=sysinit.target
